tests/migrations/test_commands.py

```diff
@@ -828,46 +829,50 @@ def test_sqlmigrate_forwards(self):
         """
         out = io.StringIO()
         call_command("sqlmigrate", "migrations", "0001", stdout=out)
-        output = out.getvalue().lower()
 
-        index_tx_start = output.find(connection.ops.start_transaction_sql().lower())
-        index_op_desc_author = output.find("-- create model author")
-        index_create_table = output.find("create table")
-        index_op_desc_tribble = output.find("-- create model tribble")
-        index_op_desc_unique_together = output.find("-- alter unique_together")
-        index_tx_end = output.find(connection.ops.end_transaction_sql().lower())
+        lines = out.getvalue().splitlines()
 
         if connection.features.can_rollback_ddl:
-            self.assertGreater(index_tx_start, -1, "Transaction start not found")
-            self.assertGreater(
-                index_tx_end,
-                index_op_desc_unique_together,
-                "Transaction end not found or found before operation description "
-                "(unique_together)",
-            )
-
-        self.assertGreater(
-            index_op_desc_author,
-            index_tx_start,
-            "Operation description (author) not found or found before transaction "
-            "start",
-        )
-        self.assertGreater(
-            index_create_table,
-            index_op_desc_author,
-            "CREATE TABLE not found or found before operation description (author)",
-        )
-        self.assertGreater(
-            index_op_desc_tribble,
-            index_create_table,
-            "Operation description (tribble) not found or found before CREATE TABLE "
-            "(author)",
-        )
-        self.assertGreater(
-            index_op_desc_unique_together,
-            index_op_desc_tribble,
-            "Operation description (unique_together) not found or found before "
-            "operation description (tribble)",
+            self.assertEqual(lines[0], connection.ops.start_transaction_sql())
+            self.assertEqual(lines[-1], connection.ops.end_transaction_sql())
+            lines = lines[1:-1]
+
+        self.assertEqual(
+            lines[:3],
+            [
+                "--",
+                "-- Create model Author",
+                "--",
+            ],
+        )
+        self.assertTrue(fnmatch(lines[3].lower(), "create table *author*"))
```

Comment:
`create table` is not a filename, why we use `fnmatch()` here? :thinking: I'd use `assertIn()`:
```suggestion
        self.assertIn(
            'CREATE TABLE %s' % connection.ops.quote_name("migrations_author"),
            lines[3],
        )
```

Reply:
I used `fnmatch` inspired by pytest’s line matcher class:

https://docs.pytest.org/en/stable/reference/reference.html#pytest.LineMatcher.fnmatch_lines

I find it a bit easier to use than a regex

you're right we could check for an absolute string instead with `assertIn`

I'll keep the lower case, since the SQL may be a different case on third party backends...